commons/Cargo.toml

```diff
@@ -0,0 +1,7 @@
+[package]
+name = "commons"
+version = "0.1.0"
+authors = ["Stefan Junker <mail@stefanjunker.de>"]
+edition = "2018"
```

Comment:
Other crates are at an older edition (they should be updated at some point), I'm not sure if that's an issue.

Reply:
This is the default now when you `cargo init ..`, so I didn't consciously set this. I'd take the chance to see if it causes trouble eventually, it hasn't as of now ;-)

Reply:
That's going to require Rust 1.31, which Red Hat doesn't support. We should drop back to 2015 for now.

Reply:
Who can bump Rust in Red Hat? It's the latest stable so I don't expect much push back. Anyway, removing this doesn't hurt anything

Reply:
I know we currently have a 1.29 requirement for coreos-metadata (see https://github.com/coreos/coreos-metadata/pull/127#issuecomment-447907284), because we need to build that as a normal rpm in a RHEL/fedora/centos buildroot.

However cincinnati is currently built in a musl container (now at 1.31) and there is no minimum version check in CI, so it is effectively already at 1.31.

Reply:
@lucab I'm pretending we actually dependon the RHEL build pipeline at one point. We only have CD from the app-sre system and not from the openshift system. AFAIU the latter will require to use official build images instead of custom ones, but we'll see ;-)